Abstract/portable-formula.rb

```diff
@@ -84,10 +84,11 @@ def self.inherited(subclass)
       on_linux do
         on_intel do
           depends_on "glibc@2.13" => :build
-          depends_on "linux-headers@4.4" => :build
         end
-        # When we move from Ubuntu 22.04, on ARM we should add a dependency to glibc@2.35 and linux-headers@5.15.
```

Comment:
Shouldn't we keep this?

Reply:
22.04 uses 2.35, though we're not exactly going to stay on 22.04 forever so would need a `glibc@whatever` for whatever we end up using.

2.17 would support every single aarch64/arm64 Linux OS out there (to our knowledge). 2.35 would only support OS versions since mid-2022, including no support at all for RHEL (their latest version uses 2.34).

Overall though I don't really mind.

Reply:
Losing RHEL for ARM specifically seems fine. I think we should aim to just support Ubuntu >=22.04 for now.

It's likely to be a long time before we're doing anything other than Portable Ruby and Homebrew/brew basic support.